mkdocs.yml

```diff
@@ -131,7 +131,7 @@ nav:
       - 'Code of conduct': 'code-of-conduct.md'
       - 'Design Docs': 'design_docs.md'
       - 'Design Doc Blueprint': 'design_doc_blueprint.md'
-      - 'Temporary Voting for Governance': 'governance/old-temporary-voting.md'
+      - 'Governance': 'governance/GOVERNANCE.md'
```

Comment:
Nit: Preserve the old link, as its still somewhat useful.

Reply:
Yeah, I preserved it. Let's just fix the docs in a small follow up (e.g. moving this old doc out of the way into a new spot.)